src/main/java/uk/nhs/hee/tis/trainee/reference/dto/CurriculumDto.java

```diff
@@ -31,4 +31,6 @@
   private String id;
   private String tisId;
   private String label;
+  private String code;
```

Comment:
This should be `curriculumSubType` rather than `code`, it's being fed by `tcs.Curriculum.curriculumSubType`